src/commands/create-apikey.ts

```diff
@@ -0,0 +1,62 @@
+import {BaseCommand} from '../lib'
+import {getEnvironment} from '../lib/environments'
+import {flags} from '@oclif/command'
+import fetch from 'node-fetch'
+import {cli} from 'cli-ux'
+
+export default class CreateApikey extends BaseCommand {
+  static description = 'Create an API key for a Backend by id'
+
+  static examples = [
+    '$ slash-graphql create-apikey "0xid"',
+  ]
+
+  static flags = {
+    ...BaseCommand.commonFlags,
+    name: flags.string({char: 'n', description: 'Client name', default: 'slash-graphql-cli'}),
+    admin: flags.boolean({char: 'a', description: 'Grant admin role', default: false})
```

Comment:
this should probably be role, and you can provide `options: ['admin', 'client']`

Reply:
That makes sense. Updating!